bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/handler/InsteonDeviceHandler.java

```diff
@@ -72,15 +72,17 @@
             InsteonBindingConstants.KEYPAD_BUTTON_E, InsteonBindingConstants.KEYPAD_BUTTON_F,
             InsteonBindingConstants.KEYPAD_BUTTON_G, InsteonBindingConstants.KEYPAD_BUTTON_H,
             InsteonBindingConstants.KWH, InsteonBindingConstants.LAST_HEARD_FROM,
-            InsteonBindingConstants.LED_BRIGHTNESS, InsteonBindingConstants.LIGHT_DIMMER,
-            InsteonBindingConstants.LIGHT_LEVEL, InsteonBindingConstants.LOAD_DIMMER,
+            InsteonBindingConstants.LED_BRIGHTNESS, InsteonBindingConstants.LED_ONOFF,
+            InsteonBindingConstants.LIGHT_DIMMER,InsteonBindingConstants.LIGHT_LEVEL, 
+            InsteonBindingConstants.LIGHT_LEVEL_ABOVE_THRESHOLD,InsteonBindingConstants.LOAD_DIMMER,
```

Comment:
space after comma

Reply:
Weird, not sure why me opening, saving in Eclipse didn't fix this. Anyway, Done